### DISCUSSION TITLE
Published an article about running and debugging a Linux UI .NET Avalonia Application using WSL and VS2022 on Windows

### DISCUSSION THREAD

**Discussion by npolyak** [AvaloniaUI/Avalonia#8040] [General]
Published an article about running and debugging a Linux UI .NET Avalonia Application using WSL and VS2022 on Windows: https://www.codeproject.com/Articles/5330464/Running-and-Debugging-Multiplatform-NET-Core-NET5

**Comment by robloo** [comment]
Any way we can get all these links collected and put in Avalonia docs? I know you mentioned in the past that was a possibility.

**Reply by npolyak** [reply]
Thanks robloo!
this question should be asked people from Avalonia. I'll try to consolidate all links on my github pages.
